src/ProjectOrigin.Vault/Database/Postgres/Scripts/v2/v2-0007.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE withdrawn_cursors (
+    stamp_name VARCHAR(256) NOT NULL,
```

Comment:
is this a name or url ? Are we sure the name is unique? Could stamp name just be stamp?

Reply:
That is a name. The plan is to have a shared network config between us, Elia and others where the config for this lives.